Example/Emission/ARDefaults.m

```diff
@@ -1,6 +1,42 @@
 #import "ARDefaults.h"
 #import <Foundation/Foundation.h>
```

Comment:
Consolidates all the logic around "should we use apphub" etc - inside a readonly package for others to use. Instead of reading the tea-leaves from build flags.

Reply:
Also simplifies the AppDelegate.

Reply:
Hah - wrong file, this is just the defaults. I added the default defaults - the registerDefaults sets fallback defaults if someone hasn't set something up yet.